CONTRIBUTING.md

```diff
@@ -1,109 +1,62 @@
-
-#### T2 - Tests Must not be Random
-
-Inputs for tests should not be generated randomly. Accounts used to create test contracts are an exception, those can be random. Also, the type and structure of outputs should be checked.
-
-
-### Documentation
-
-TODO
-
-## Pull Request Workflow
+```
+cd my-zeppelin-solidity-fork
+git fetch upstream
+git checkout development
+git pull --rebase upstream development
+```
 
-Our workflow is based on GitHub's pull requests. We use feature branches, prepended with: `test`, `feature`, `fix`, `refactor`, or `remove` according to the change the branch introduces. Some examples for such branches are:
-```sh
-git checkout -b test/some-module
-git checkout -b feature/some-new-stuff
-git checkout -b fix/some-bug
-git checkout -b remove/some-file
+2) Branch out from `development` into `fix/some-bug-#123`:
+(Postfixing #123 will associate your PR with the issue #123 and make everyone's life easier =D)
+```
+git checkout -b fix/some-bug-#123
 ```
 
-If your branch is planned to fix an open issue, postfix your branch name with the issue number like `fix/some-bug-#123`.
+3) Make your changes, add your files, commit and push to your fork.
 
-We expect pull requests to be rebased to the master branch before merging:
-```sh
-git remote add zep git@github.com:OpenZeppelin/zeppelin-solidity.git
-git pull --rebase zep master
+```
+git add -A
+git commit "Fix some bug #123"
+git push origin fix/some-bug-#123
 ```
 
-Note that we require rebasing your branch instead of merging it, for commit readability reasons.
+4) Go to [github.com/OpenZeppelin/zeppelin-solidity](https://github.com/OpenZeppelin/zeppelin-solidity) in your web browser and issue a new pull request.
 
-After that, you can push the changes to your fork, by doing:
-```sh
-git push origin your_branch_name
-git push origin feature/some-new-stuff
-git push origin fix/some-bug
-```
+*IMPORTANT* Read the PR template very carefully and make sure to follow all the instructions. These instructions 
+refer to some very important conditions that your PR must meet in order to be accepted, such as making sure that all tests pass, JS linting tests pass, solidity linting tests pass, etc.
+
+5) Maintainers will review your code and possibly ask for changes before your code is pulled in to the main repository. We'll check that all tests pass, review the coding style, and check for general code correctness. If everything is OK, we'll merge your pull request and your code will be part of Open Zeppelin.
 
-Finally go to [github.com/OpenZeppelin/zeppelin-solidity](https://github.com/OpenZeppelin/zeppelin-solidity) in your web browser and issue a new pull request.
+*IMPORTANT* It is not uncommon for almost finished PRs to stay unmerged for a long time, so, considering the effort you put in your code, please pay attention to the maintainer's feedback since its a necessary to keep up with the standards Open Zeppelin attains to. 
```

Comment:
Though I like the warning, I feel like it's too negative. I'd remove this, and just add the "Please pay attention to the maintainer's feedback since its a necessary to keep up with the standards Open Zeppelin attains to" to the previous paragraph.